multigeiger/multigeiger.ino

```diff
@@ -260,8 +288,9 @@ void loop() {
   set_status(STATUS_HV, hv_error ? ST_HV_ERROR : ST_HV_OK);
 
   int wifi_status = update_wifi_status();
+  update_bt_status();
```

Comment:
looks like we have some `bt` and some `ble` now.

Reply:
yes, indeed, I always used ble in my code, but saw your bt in the main .ino and tried to stay consistent - but wasn't of course... Would it be okay to change everything to ble? We are using Bluetooth Low Energy, and 3 letters hint more than 2.

Reply:
Guess both bt and ble is fine. One is more general, one more specific.

Reply:
to resolve this: make a ticket and a separate PR. it is a cosmetic cleanup, should not be in here.

Reply:
#315